com.microsoft.mrtk.core/Utilities/CameraFOVChecker.cs

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit
+{
+    /// <summary>
+    /// Camera extension methods to test if colliders are within camera's FOV. Uses caching to improve performance and
+    /// ensure values are only computed once per frame.
+    /// </summary>
+    public static class CameraFOVChecker
+    {
+        private static int inFOVLastCalculatedFrame = -1;
+
+        /// <summary>
+        /// Map from object => is the object in the FOV for this frame.
+        /// </summary>
+        /// <remarks>
+        /// This map is cleared every frame.
+        /// </remarks>
+        private static readonly Dictionary<ValueTuple<Collider, Camera>, bool> inFOVColliderCache = new Dictionary<ValueTuple<Collider, Camera>, bool>();
```

Comment:
I _think_ you should be able to do this too?

```suggestion
        private static readonly Dictionary<(Collider, Camera), bool> inFOVColliderCache = new Dictionary<(Collider, Camera), bool>();
```